### PR TITLE
Fixed dependency injection

### PR DESCRIPTION
**Pull request by muffins90**

### CODE REVIEW

**Comment by dmach** [comment]
Can you change 'rel' to 'release'?

File: releng_sop/koji_block_package_in_release.py [hunk 1]
```diff
@@ -16,13 +16,18 @@
...
-from .common import Environment
+from .common import Environment, Release
 from .kojibase import KojiBase
 
 
 class KojiBlockPackageInRelease(KojiBase):
     """Block packages in a release."""
 
+    def __init__(self, env, rel, packages):
+        """Adding packages for block."""
```
Comment (csomh) [inline releng_sop/koji_block_package_in_release.py]: Adding packages *to* block.

File: releng_sop/koji_block_package_in_release.py [hunk 2]
```diff
@@ -16,13 +16,18 @@
 
 import argparse
 
-from .common import Environment
+from .common import Environment, Release
 from .kojibase import KojiBase
 
 
 class KojiBlockPackageInRelease(KojiBase):
     """Block packages in a release."""
```
Comment (csomh) [inline releng_sop/koji_block_package_in_release.py]: All constructor parameters should be documented here (eng, release, packages).

File: releng_sop/koji_clone_tag_for_release_milestone.py [hunk 3]
```diff
@@ -19,29 +19,26 @@
...
 from .common import Environment, Release
+from .kojibase import KojiBase
 
 
-class KojiCloneTagForReleaseMilestone(object):
+class KojiCloneTagForReleaseMilestone(KojiBase):
     """
     Clone tag for release milestone.
 
-    :param env: name of the environment to be used to execute the commands.
```
Comment (csomh) [inline releng_sop/koji_clone_tag_for_release_milestone.py]: We should keep documentation for `env` and `release` here.

File: releng_sop/koji_create_package_in_release.py [hunk 4]
```diff
@@ -16,16 +16,19 @@
 
 import argparse
 
-from .common import Environment
+from .common import Environment, Release
 from .kojibase import KojiBase
 
 
 class KojiCreatePackageInRelease(KojiBase):
     """Create packages in a release."""
```
Comment (csomh) [inline releng_sop/koji_create_package_in_release.py]: Docs for constructor arguments :smiley:

**Pull request merged**